components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -178,6 +178,14 @@
     public static final String GOVERNANCE_COMPONENT_REGISTRY_LOCATION = "/repository/components/org.wso2.carbon" +
             ".governance";
 
+    // registry location for OpenAPI files
+    public static final String OPENAPI_ARCHIVES_TEMP_FOLDER = "OPENAPI-archives";
+    public static final String OPENAPI_EXTRACTED_DIRECTORY = "extracted";
+    public static final String OPENAPI_ARCHIVE_ZIP_FILE = "openapi-archive.zip";
+    public static final String CHARSET = "UTF-8";
+    public static final String OPENAPI_MASTER_JSON = "/swagger.json";
```

Comment:
Shall we omit '/' from the file name?

Reply:
https://github.com/wso2/carbon-apimgt/pull/8509/commits/bbdcfe5dd9f994b5693c99f127caa4948e5f3f87#diff-d28b77ae984cf341809858ec0fb93f3fR186